uint64 timestamp
std_msgs/Float32MultiArray descriptors
std_msgs/Float64MultiArray reconstructions